__tests__/index.js

```diff
@@ -38,3 +38,91 @@ describe('Testing idempotence of groupBy and oldGroupBy', () => {
     expect(groupBy(arr, fieldName)).toEqual(oldGroupBy(arr, fieldName));
   });
 });
+
+describe('resolvePath', () => {
+  it('should return nested value', () => {
+    expect(resolvePath({ a: { b: 'c' } }, 'a.b')).toEqual('c');
+  });
+  it('should return undefined for wrong path', () => {
+    expect(resolvePath({ a: { c: 'c' } }, 'a.b.c')).toEqual(undefined);
+  });
+});
+
+describe('sortByFields', () => {
+  const array = [
+    {
+      line: 1,
+      page: 1,
+      test: 'a',
+    },
+    {
+      line: 2,
+      page: 1,
+      test: 'A',
+    },
+    {
+      line: 4,
+      page: 1,
+      test: 2,
+    },
+    {
+      line: 3,
+      page: 2,
+      test: 'B',
+    },
+    {
+      line: 4,
+      page: 1,
+      test: 1,
+    },
+  ];
+
+  it('should return sort by page ASC', () => {
+    expect(sortByFields(array, ['page', 'ASC'])).toMatchSnapshot();
+    expect(sortByFields(array, 'page')).toMatchSnapshot();
```

Comment:
Here you're making 2 snapshots where we expect the same value. Maybe it could be done by checking the snapshot for the first one, and then checking that the second is equal to the first.